Java/tests/src/test/java/com/microsoft/recognizers/text/tests/AbstractTest.java

```diff
@@ -265,7 +263,6 @@ public static void assertResultsWithKeys(TestCase currentCase, List<ModelResult>
         ObjectMapper mapper = new ObjectMapper();
         mapper.configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES, true);
         mapper.addMixIn(ModelResult.class, ModelResultMixIn.class);
-        mapper.addMixIn(ExtendedModelResult.class, ExtendedModelResultMixIn.class);
```

Comment:
What happened with the `ExtendedModelResultMixIn` class? Can you review if we need to remove it or not? The `ExtendedModelResult` was removed

Reply:
Please, let me know about this, I want to know if we need to remove the class, or if we need to apply another change for this removal

Reply:
Applying this [change](https://github.com/southworks/Recognizers-Text/pull/376/files#diff-b3c211fd72ec7940e1654281c36961948b46dc6445a0485f7a2da87a08924332R13) requires to update this as follows:
`mapper.addMixIn(ModelResult.class, ModelResultMixIn.class);`